README.md

```diff
@@ -33,7 +37,22 @@ The mock method takes three arguments:
 
 The mock method executes insert, then update, and finally findOne and returns the result of findOne.
 
-Each argument is optional and we have some smart behavior to handle non-existant arguments
+Each argument is optional and we have some smart behavior to handle non-existant arguments:
 - `insert`, if not specified we'll grab the first part of the `update` operator
 - `update`, if not specified we'll skip the update, if a single argument is specified, we'll use the id returned by our insert operation as the first argument (the query)
-- `find`, if not specified we'll use the id returned by our insert operation
\ No newline at end of file
+- `find`, if not specified we'll use the id returned by our insert operation
+
+**The `mockMulti` method takes four arguments:**
+
+- `insert` an array of arguments to be passed to the mockCollection's insert method
+- `update` an array of arguments to be passed to the mockCollection's update method
+- `flags` an object to be passed to the mockCollection's update method
+- `find` an array of arguments to be passed to the mockCollection's find method
+
+The mockMulti method executes insert, then update, and finally find and returns the result of find().fetch().
+
+Each argument is optional:
+- `insert`, if not specified we'll grab the first part of the `update` operator
+- `update`, if not specified we'll skip the update, if a single argument is specified, we'll use the ids returned by our insert operation as the first argument (the query)
```

Comment:
In other words, we want to make the developer explicitly specify the query for the update operation, because that's the whole point of the mockMulti (updating multiple docs which match a query).